deploy/charts/cert-manager/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: cert-manager
 # The version and appVersion fields are set automatically by the release tool
-version: v0.1.0
-appVersion: v0.1.0
+version: v1.3.0
+appVersion: v1.3.0
```

Comment:
This  directory contains only the template of a Helm chart, which is is processed by the `bazel` based build scripts.
It is not designed to be installed directly.

See https://cert-manager.io/docs/installation/kubernetes/#installing-with-helm for information about installing cert-manager using the official Helm chart.

And as hinted at by the comment above, 
these versions are place holder values only. They are substituted by the build scripts. 



Here's how you might build a working helm chart from source:

```
bazel build //deploy/charts/cert-manager:release-tar 
tar xf  bazel-bin/deploy/charts/cert-manager/release-tar.tar.gz
helm show all chart/cert-manager.tgz  | head
```